lib/jazzy/sourcekitten.rb

```diff
@@ -320,14 +320,18 @@ def self.deduplicate_declarations(declarations)
     def self.deduplication_key(decl)
       if decl.type.swift_extensible? || decl.type.swift_extension?
         [decl.usr, decl.name]
+      elsif decl.type.objc_class? || decl.type.objc_category?
+        name, = decl.objc_category_name || decl.name
+        [name, 'objc_class_and_categories']
       else
         [decl.usr, decl.name, decl.type.kind]
       end
     end
 
     # Merges all of the given types and extensions into a single document.
+    # rubocop:disable Metrics/MethodLength
```

Comment:
this should either be re-enabled after the method, or this should go on the same line as the method declaration